src/Culling/babylon.ray.ts

```diff
@@ -216,8 +216,47 @@
 
         }
 
+        public intersectsMeshes(meshes:Array<AbstractMesh>, fastCheck?: boolean, results?:Array<PickingInfo>): Array<PickingInfo> {
+
+            if(results){
+                results.length = 0;
+            }else{
+                results = [];
+            }
+
+            for(var i = 0; i < meshes.length; i++){
+                var pickInfo = this.intersectsMesh(meshes[i], fastCheck);
+
+                if(pickInfo.hit){
+                    results.push(pickInfo);
+                }
+            }
+
+            results.sort(this._comparePickingInfo);
+
+            return results;
+
+        }
+
+        private _comparePickingInfo(pickingInfoA:PickingInfo, pickingInfoB:PickingInfo): number{
+
+            if(pickingInfoA.distance < pickingInfoB.distance){
+                return -1;
+            }else if(pickingInfoA.distance > pickingInfoB.distance){
+                return 1;
+            }else{
+                return 0;
+            }
+
+        }
+
+        /**
+         *  @Deprecated. Use new RayHelper.show() instead.
+         * */
         public show(scene:Scene, color:Color3): void{
 
+            console.warn('Ray.show() has been deprecated.  Use new RayHelper.show() instead.');
+

```

Comment:
is it possible to then use RayHelper internally as well to avoid code duplication?